### DISCUSSION TITLE
Evaluate ExUnits get the error when try to unlock locked fund

### DISCUSSION THREAD

**Discussion by quanqx** [bloxbean/cardano-client-lib#386] [Q&A] [answered]
Hi,
I'm trying to create a simple project to lock & unlock fun.
I follow the guild, I create my own smart contract by aiken and then try to interact with it but can not unlock the fund.
I try to post the error to Cardano forum but there is no answer.
Detail error: https://forum.cardano.org/t/cardano-client-lib-fail-to-evaluate-ex-units-when-send-the-locked-fund/129752
Could you please help to take a look?
Thank you so much!

**Comment by satran004** [comment, marked as answer]
Hi @quanqx,

You need to set requiredSigners field in transaction as the contract is checking signatories.

Here's a working example with QuickTx

https://github.com/satran004/helloworld-contract-offchain 

Looks like you are using Composable function (TxBuilder) api in your off-chain code. Check QuickTx api, it provides a simple declarative api to build transactions.

Thanks.

**Reply by quanqx** [reply]
Thank you! I solved the problem using QuickTx.
